modules/schematics/src/entity/creator-files/__name@dasherize@if-flat__/__name@dasherize@group-reducers__.reducer.ts.template

```diff
@@ -49,9 +49,11 @@ export const reducer = createReducer(
   ),
 );
 
+export const <%= pluralize(name) %>Feature = createFeatureSelector<State>(<%= pluralize(name) %>FeatureKey);
```

Comment:
I'm not a big fan of using `createFeatureSelector` (and selectors in general) in a reducer file.
The only exception is `createFeature`.
Leaving this open so someone else can give their opinion.

Reply:
It does seem to make the file have multiple responsibilities, but you cant have a reducer without selectors, I think of them as getters and setters for the state so, in my opinion, I always put them in the same file

Reply:
I'm in agreement with Tim on this one. Feature selectors should be connected elsewhere as it makes the reducer file more portable on its own.

Reply:
The only purpose of the feature selector is to be passed to the adapter's method so the typing is corrected, otherwise the selectors wont properly be typed.
Feel free to make a decision regarding this pull request, thought this would be a positive addition to this library.

Reply:
@PixelatedFigure AFAIK, entity selectors with or without passed feature selector will be properly typed.

I think we can close this PR and move the discussion to #3270.